analyze_ipa.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"archive/zip"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path/filepath"
+)
+
+const (
+	contentsFolder = "contents"
+	payloadFolder  = contentsFolder + "/Payload"
```

Comment:
[filepath.Join](https://golang.org/pkg/path/filepath/#Join) is a safe way to join file path elements